sark/code/line.py

```diff
@@ -144,6 +144,11 @@ def is_data(self):
         return idaapi.isData(self.flags)
 
     @property
+    def is_string(self):
+        """Is the line data of type ASCII string."""
+        return idaapi.isASCII(self.flags)
```

Comment:
This implementation is problematic as it only addresses ASCII strings.
I think a better solution is to do something along the lines of `get_string` https://github.com/tmr232/Sark/blob/master/sark/data.py#L118